src/network/networkprotocol.h

```diff
@@ -491,7 +494,7 @@ enum ToClientCommand
 		u8[len] formspec
 	*/
 
-	TOCLIENT_DETACHED_INVENTORY = 0x43,
+	TOCLIENT_DETACHED_INVENTORY = 0x,
```

Comment:
rebase mistake (how?)

Reply:
oops! fixed. (how: i was lazy and did the edits in GH's web editor. i searched for 43 in case it was mentioned / hardcoded elsewhere in the file; must've accidentally hit replace)